### DISCUSSION TITLE
How to apply gradient conditional formatting?

### DISCUSSION THREAD

**Discussion by and7ey** [nithinmurali/pygsheets#592] [Q&A]
I would like to use conditional formatting in my doc - 
<img width="94" alt="image" src="https://github.com/nithinmurali/pygsheets/assets/900474/6c14400a-2a2d-4df4-a1bf-a7b67918b628">
(https://workspaceupdates.googleblog.com/2015/03/conditional-formatting-improvements-in.html)

There is no GRADIENT condition type - https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/other#ConditionType. I tried to use CUSTOM_FORMULA - https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/other#ConditionType - but it didn't help (or I used it incorrectly).

What else should I try?

**Comment by nithinmurali** [comment]
Looks like this example applies conditional formatting with gradient?
https://developers.google.com/sheets/api/samples/conditional-formatting#color-gradient

**Reply by and7ey** [reply]
The question is how to repeat it with pygsheets
